bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -211,6 +211,8 @@ spec:
           - namespaces
           verbs:
           - get
+          - list
```

Comment:
Please squash this into the commit the kubebuilder directives are being added.
Else this will cause revert issues if one of the commits needs to be reverted